src/modules/getArticles.js

```diff
@@ -2,18 +2,23 @@ import axios from "axios";
 
 const getArticles = {
   async index(dispatch) {
-    let result = await axios.get("/articles");
-    dispatch({type: "SET_NEWS_FEED", payload: result.data.articles})
+    const result = await axios.get("/articles");
+    dispatch({ type: "SET_NEWS_FEED", payload: result.data.articles });
   },
 
   async show(articleId, dispatch) {
     try {
-      let response = await axios.get(`/articles/${articleId}`);
+      const response = await axios.get(`/articles/${articleId}`);
       dispatch({ type: "VIEW_ARTICLE", payload: response.data.article });
     } catch (error) {
       dispatch({ type: "ERROR_MESSAGE", payload: error.response.data.message });
     }
   },
+
+  async index_by_category(categoryId, dispatch) {
```

Comment:
are you really finding the category by the id in the api?

Reply:
We were using label, so we fixed it in the api!